Feature/bootsrap layout

SerbanGabriel opened the pull request:
Implemented bootstrap layout for admin-console in Heimdall.

SerbanGabriel committed:
Implemented bootstrap layout for loginpage

SerbanGabriel committed:
code cleanup

SerbanGabriel committed:
implemented boostrap for home

SerbanGabriel committed:
fixing css issue on home page

SerbanGabriel committed:
implemented bootstrap navbar

SerbanGabriel committed:
finsihed implementing bootstrap layoyut for home component

SerbanGabriel committed:
small changes

SerbanGabriel committed:
realm button width change

SerbanGabriel committed:
implemented bootstrap layout for client component

SerbanGabriel committed:
implemented bootstrap layout for users component

SerbanGabriel committed:
implemented bootstrap layout for roles component

SerbanGabriel committed:
Merge branch 'master' into feature/bootsrap-layout

SerbanGabriel committed:
implemented boostrap layout for role component

SerbanGabriel committed:
Merge branch 'master' into feature/bootsrap-layout

kis-robert commented on src/app/components/clients/clients.component.css:
```diff
@@ -1,48 +1,8 @@
-.edit:hover {
-    background-color: lightblue;
-    color: white;
+.my-icon {
```
fara my-icon, example-icon, pune un nume mai sugestiv

kis-robert replied:
fara my-icon, example-icon, pune un nume mai sugestiv

kis-robert commented on src/app/components/clients/clients.component.css:
```diff
@@ -1,48 +1,8 @@
-.auth {
-    font-size: 50px;
+.my-icon:hover {
+    color: rgb(32, 66, 218);
```
exista https://developer.mozilla.org/en-US/docs/Web/CSS/var pentru culori, iti definesti culorile undeva si poti refolosi aceeasi nuanta prin variabila

kis-robert commented on src/app/components/heimdall-roles/heimdall-roles.component.css:
```diff
@@ -1,59 +1,18 @@
-    flex-direction: column-reverse;
-    align-items: center;
+.my-table {
+    width: 300px;
```
fara valori absolute, nu va arata bine pe responsive, foloseste grid-ul de la bootstrap

kis-robert commented on src/app/components/heimdall-roles/heimdall-roles.component.css:
```diff
@@ -1,59 +1,18 @@
-.auth {
-    font-size: 50px;
-    text-align: center;
+@media only screen and (max-width: 350px) {
```
daca te folosesti de bootstrap nu ar trebui sa ai probleme de genul asta

kis-robert commented on src/app/components/heimdall-roles/heimdall-roles.component.html:
```diff
@@ -1,26 +1,39 @@
+<div class="container-fluid">
+    <div class="row m-5">
+        <div class="col d-flex justify-content-end">
+            <mat-form-field class="example-form-field mr-5">
```
fara example

kis-robert commented on src/app/components/heimdall-roles/heimdall-roles.component.html:
```diff
@@ -1,26 +1,39 @@
+            </mat-form-field>
+        </div>
+        <div class="col d-flex align-items-center">
+            <button class="mr-3" mat-raised-button (click)="addRole(role.value)" color="primary">New Role</button>
```
ar trebui sa folosesti un formular pentru Add, cu validari

kis-robert commented on src/app/components/heimdall-roles/heimdall-roles.component.html:
```diff
@@ -1,26 +1,39 @@
+        </div>
+    </div>
+    <div class="row">
+        <div class="col d-flex justify-content-center">
```
Typo? Ce reprezinta 'col' simplu?

kis-robert commented on src/app/components/login/login.component.css:
```diff
@@ -1,125 +1,69 @@
+            }
+        }
+        
+        .errormessaje {
```
engleza, iar pentru faza cu color red, cred ca ai clasa text-danger in bootstrap

kis-robert commented on src/app/components/login/login.component.ts:
```diff
@@ -11,7 +11,7 @@ import { AdminAuthService } from '../../services/admin-auth/admin-auth.service';
 })
 
 export class LoginComponent implements OnInit {
-  errormessaje:boolean = false;
```
engleza

kis-robert commented on src/app/components/users/users.component.html:
```diff
@@ -1,34 +1,43 @@
+            </mat-form-field>
+        </div>
+        <div class="col col-lg-2 d-felx  align-items-center">
+            <button [disabled]="password === '' || username === ''" (click)="addUser(username.value,password.value)" mat-raised-button color="primary">Add</button>
```
validare prin formular

kis-robert commented on src/index.html:
```diff
@@ -8,11 +8,15 @@
 
 <body>
     <app-root></app-root>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```
cred ca varianta corecta e sa adaugi in angular.json in array-ul de style si script

kis-robert reviewed (changes requested):
Fa modificarile si anunta-ma sa ma uit din nou peste ele.

SerbanGabriel committed:
--implemented global variable for color
-renamed most of classes
-removed absolute values
-removed @media only screen
-removed example classes
-fixing typos
-adding form validation

SerbanGabriel committed:
removed extra bootstrap stylesheet and script from index.html

SerbanGabriel committed:
removed console.log

kis-robert commented on src/app/components/login/login.component.css:
```diff
@@ -43,10 +43,6 @@
-            color: red;
-        }
-        
         .my-container h1 {
```
fara my-container

kis-robert commented on src/app/components/login/login.component.css:
```diff
@@ -43,10 +43,6 @@
-        
         .my-container h1 {
             font-size: 4vw;
             background: -webkit-linear-gradient(rgb(202, 190, 228), rgba(44, 41, 214, 0.753));
```
background cu variabile pentru culori

kis-robert commented on src/app/components/users/users.component.html:
```diff
@@ -1,19 +1,22 @@
+                    <input matInput formControlName="username">
+                </mat-form-field>
+
+                <div class="col col-lg-2 d-flex justify-content-center ">
```
typo

kis-robert commented on src/app/components/users/users.component.ts:
```diff
@@ -28,12 +33,16 @@ export class UsersComponent implements OnInit {
   }
 
+  onSubmit(){
+    this.addUser(this.form.value.username,this.form.value.password)
```
trimite tot obiectul nu pe bucati

kis-robert commented on src/styles.css:
```diff
@@ -1,6 +1,11 @@
+
 @import "~@angular/material/prebuilt-themes/indigo-pink.css";
-*{
+* {
```
scoate asta, de ce sa pui pe toate elementele 0 margin / padding?

SerbanGabriel committed:
removed default margin 0 and padding 0

SerbanGabriel committed:
changed the name of my-container

SerbanGabriel committed:
changed onSubmit on user component

SerbanGabriel committed:
removed console.log

SerbanGabriel committed:
changed sidenav content container

SerbanGabriel committed:
changed all columns from components
